### PR TITLE
PR for to ensure native modules are compiled with java version 8

### LINKED ISSUE #14692: bump java version to 8 research issue
we have this issue in place to research what it would take to upgrade to java version 8.
We are currently on java version 7.

### PR DESCRIPTION
**Pull request by siddarthkay**
fixes: #14692
status: ready

### CODE REVIEW

**Review by bitgamma — APPROVED** [review]

**Comment by siddarthkay** [comment]
@status-im/mobile-qa : This PR should not require manual testing since the only change that this PR does is to ensure that native modules are compiled with java version 8, they were previously compiled with java version 7.

**Comment by siddarthkay** [comment]
> @status-im/mobile-qa : This PR should not require manual testing since the only change that this PR does is to ensure that native modules are compiled with java version 8, they were previously compiled with java version 7.

@status-im/mobile-qa : however do let me know if you would like to test it though :)

**Review by J-Son89 — APPROVED** [review]

**Comment by churik** [comment]
not really, e2e should be enough :)

File: modules/react-native-status/android/build.gradle [hunk 1]
```diff
@@ -19,6 +19,11 @@ def getStatusGoSHA1 = { ->
 android {
     compileSdkVersion rootProject.ext.compileSdkVersion
 
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+        }
```
Comment (jakubgs) [inline modules/react-native-status/android/build.gradle]: Wrong indent on closing bracket.

**Review by jakubgs — APPROVED** [review]

**Pull request merged**